.github/workflows/chart-release.yaml

```diff
@@ -85,9 +96,26 @@ jobs:
         run: |
           echo "chart_version=${{ inputs.chart_version }}" >> $GITHUB_ENV
 
+      - name: Determine Image Tag
+        run: |
+          if [[ "${{ inputs.tag_generation }}" == 'ticket_from_branch' ]]; then
+            regex='^([[:alpha:]]+?-[[:digit:]]+)'
+            if [[ $GITHUB_REF_NAME =~ $regex ]]; then
+              image_tag="${BASH_REMATCH[1]}"
+            else
+              echo "::error::GITHUB_REF_NAME does not match the expected pattern for ticket_from_branch."
```

Comment:
Maybe a error message like "Couldn't extract ticket from branch $GITHUB_REF_NAME. The branch name should begin with a ticket (e.g. SPSH-1234-test-name)" would be easier to understand for someone without deeper knowledge about the specific implementation.